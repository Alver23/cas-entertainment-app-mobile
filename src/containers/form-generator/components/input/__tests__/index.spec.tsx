// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

// Components
import { InputForm } from '../index';

const MemoizeComponent = InputForm.type;

describe('InputForm', () => {
  let component: ShallowWrapper;

  const props = {
    placeholder: 'fake placeholder',
    onChangeText: jest.fn(),
    onBlur: jest.fn(),
    value: 'fake value',
    errorMessage: 'fake mssage',
  };

  beforeEach(() => {
    component = shallow(<MemoizeComponent {...props} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
