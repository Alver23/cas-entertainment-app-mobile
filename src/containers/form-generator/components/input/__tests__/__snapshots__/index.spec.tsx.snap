// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputForm should save a snapshot of the component 1`] = `
<ForwardRef(Themed.Input)
  errorMessage="fake mssage"
  inputContainerStyle={
    Array [
      Object {
        "borderColor": "red",
      },
    ]
  }
  onBlur={[MockFunction]}
  onChangeText={[MockFunction]}
  placeholder="fake placeholder"
  value="fake value"
/>
`;
